docs/terraform.rst

```diff
@@ -15,11 +15,31 @@ AWS resources like Cognito user pools.
     Please step cautiously and be careful when using them!
 
 
+Configurations
+==============
+
+All ``terraform`` commands below expect to be run from within a directory
+containing a Terraform configuration (i.e. a set of one or more ``*.tf`` or
+``*.tf.json`` files).
+
+We have the following configurations:
+
+  - :file:`env/production`
+
```

Comment:
`env/testing` is missing from this list.

Reply:
Thanks for the catch!

Reply:
https://github.com/nextstrain/nextstrain.org/pull/661